Gemfile.lock

```diff
@@ -1,3 +1,22 @@
+GIT
+  remote: git@github.com:wordpress-mobile/release-toolkit.git
+  revision: 908ff74c45a3ccd022adb028b2b4a2fad296e27a
+  ref: 908ff74c45a3ccd022adb028b2b4a2fad296e27a
```

Comment:
See [comment in the `fastlane/Pluginfile` file](https://github.com/wordpress-mobile/WordPress-Android/pull/15282/files#diff-fc04a28e75ab5e6e1aeb7ac7f0759e9c9d7ef3d650e581c0f355f0e4cfeae9e8R9).

TL;DR: it's expected that we're pointing on a ref and not on a stable version of the toolkit here.